prep-javascript-objects/main.js

```diff
@@ -0,0 +1,19 @@
+var person = {
+  firstName: 'John',
+  lastName: 'Smith',
+  hobby: 'baseball',
+  job: 'plumber',
+  previousJob: 'teacher'
```

Comment:
We don't want to set these here, they should be set later in the code using dot notation. The idea is to learn different ways of setting a property on an object.